appdatabase/migrations/sql/1702651464_community_mailservers.up.sql

```diff
@@ -0,0 +1,13 @@
+-- To allow many to many relationship between communities and mailservers
+CREATE TABLE IF NOT EXISTS community_mailservers (
+    community_id BLOB NOT NULL,
+    mailserver_id VARCHAR NOT NULL,
```

Comment:
Since these can change overtime (with new storenodes added or removed), I wonder if it would make sense to add a `clock` and `removed` column to handle these type of changes? This would be similar to how bookmarks are managed in protocol/messenger_bookmarks.go